--- constants.rs	2022-07-20 23:12:00.248049049 +0200
+++ patch_constants_snap_hostfs_prefix.rs	2022-07-20 23:50:09.282598492 +0200
@@ -1,5 +1,5 @@
 // External
-pub(crate) const INPUTFILES_PATH_PREFIX: &str = "/";
+pub(crate) const INPUTFILES_PATH_PREFIX: &str = "/var/lib/snapd/hostfs";
 
 // Error messages
 pub(crate) const ERROR_OPEN_INPUT_FILE_: &str = "An error occurred while trying to open the input file: ";
